packages/dx-react-chart/src/chart.tsx

```diff
@@ -16,7 +16,15 @@ import { Palette } from './plugins/palette';
 import { Root } from './templates/layout';
 import { Label } from './templates/label';
 
-class RawChart extends React.PureComponent {
+const defaultProps = {
+  height: 500,
+};
+
+type RawChartDefaultProps = Readonly<typeof defaultProps>;
+
+class RawChart extends React.PureComponent<RawChartProps & RawChartDefaultProps> {
+  static defaultProps = defaultProps;
```

Comment:
Try the following (maybe there is no need to have separate `defaultProps` constant):
```js
static defaultProps = {
  height: 500,
};
```